app/controllers/items_controller.rb

```diff
@@ -693,16 +692,16 @@ def save_and_reindex
   # Permissions
 
   # check that the user can carry out this item modification
-  def forbid_modify
-    return true if current_user.is_admin || @object.can_manage_content?(current_user.roles(@apo))
-    render :status => :forbidden, :text => 'forbidden'
-    false
+  def authorize_manage_obj!
+    authorize! :manage_item, @object
   end
```

Comment:
Thanks @LynnMcRae. This distinction is already present in the code, but it would be great if we could collapse reduce the various permutations.